### PR TITLE
Add common libs for yql plugin. YT git part.

### PR DESCRIPTION
**Pull request by Tony-Romanov**

### CODE REVIEW

File: yt/yql/agent/config.h [hunk 1]
```diff
@@ -129,6 +129,8 @@ class TYqlPluginConfig
 
     NYTree::INodePtr OperationAttributes;
 
+    NYTree::INodePtr Libraries;
```
Comment (gritukan) [inline yt/yql/agent/config.h:132]: Let's make it `THashMap<TString, TString>` because this field is strongly typed.

**Comment by Tony-Romanov** [comment]
https://github.com/ydb-platform/ydb/issues/9398

File: yt/yql/agent/config.cpp [hunk 2]
```diff
@@ -263,6 +263,9 @@ void TYqlPluginConfig::Register(TRegistrar registrar)
         .DefaultNew();
     registrar.Parameter("enable_dq", &TThis::EnableDq)
         .Default(false);
+    registrar.Parameter("libraries", &TThis::Libraries)
+        .Default({})
```
Comment (gritukan) [inline yt/yql/agent/config.cpp:267]: There is no need for `{}` here. Just `.Default()`.

File: yt/yql/agent/config.cpp [hunk 3]
```diff
@@ -263,6 +263,9 @@ void TYqlPluginConfig::Register(TRegistrar registrar)
...
+        .ResetOnLoad();
```
Comment (gritukan) [inline yt/yql/agent/config.cpp:268]: Why do we need ResetOnLoad here?

**Review by gritukan — COMMENTED** [review]
LGTM besides cosmetics.

Need to promote YDB commit when corresponding PR in YDB will be merged.

**Review by Kontakter — APPROVED** [review]
To run tests in internal CI

**Comment by Kontakter** [comment]
@Krisha11 – could you read and merge this PR?

**Comment by Krisha11** [comment]
LGTM, ready to merge

But you should promote YDB commit in ytsaurus CI in future PR

**Pull request closed without merging**